analysis/import/SonarImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/sonar/model/Version.kt

```diff
@@ -0,0 +1,23 @@
+package de.maibornwolff.codecharta.importer.sonar.model
+
+import de.maibornwolff.codecharta.importer.sonar.SonarImporterException
+import java.lang.Exception
+
+class Version(val major: Int, val minor: Int) {
+
+    companion object {
+        fun parse(version: String): Version {
+            try {
+                val tokenized = version.split(".").subList(0, 2)
+                val versions = tokenized.map { it.toInt() }
+                return Version(versions[0], versions[1])
+            } catch (exception: Exception) {
+                throw SonarImporterException("Could not parse version $version.")
```

Comment:
Please do not change the actual exception or at least log the actual exception. Otherwise debugging is not nice at all. Please keep that in mind as a general rule and not only for this code part. The same applies to the code that a try catch surrounds.

Reply:
PTAL